cmd/drand-cli/control.go

```diff
@@ -15,6 +17,27 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+func isWeak(password string) bool {
+	if len(password) < 12 {
+		return true
+	}
+	if strings.Contains(strings.ToLower(password), "drand") {
+		return true
+	}
```

Comment:
This is too ad-hoc... if we put this we should put we're not checking many other wild-guesses..

Reply:
Also, the user may provide a password that is perfectly valid per the documented rules and it still fails because it matches `drand`. It will be very confusing.